src/devices/fineoffset.c

```diff
@@ -380,12 +387,14 @@ static int fineoffset_WH0290_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     uint8_t id        = b[1];
     int pm25          = (b[2] & 0x3f) << 8 | b[3];
     int pm100         = (b[4] & 0x3f) << 8 | b[5];
-
+    int battery_bars  = (b[2] & 0x40) >> 4 | (b[4] & 0xC0) >> 6; //out of 5
+    float battery_ok  = battery_bars / 5.0; //convert out of 5 bars to 0 (0 bars) to 1 (5 bars)
 
     /* clang-format off */
     data = data_make(
             "model",            "",             DATA_STRING, _X("Fineoffset-WH0290","Fine Offset Electronics, WH0290"),
             "id",               "ID",           DATA_INT,    id,
+            "battery",          "Battery Level",  DATA_FORMAT, "%.1f", DATA_DOUBLE, battery_ok,
```

Comment:
should be "battery_ok", "battery" is assumed to be "OK" / "LOW"

Reply:
Fix committed.